cg/services/order_validation_service/workflows/rna_fusion/constants.py

```diff
@@ -0,0 +1,10 @@
+from enum import Enum
+
+from cg.constants.constants import DataDelivery
+
+
+class RnaFusionDeliveryType(Enum):
```

Comment:
We allow more options than these:

![image](https://github.com/user-attachments/assets/7f7943e4-ffc2-4a4d-872e-0d04f67692a7)